test-result-summary-client/src/Build/Output/Output.jsx

```diff
@@ -50,7 +50,7 @@ export default class Output extends Component {
                 buildUrl: dataInfo.buildUrl,
             };
         } else {
-            const results = await fetchData(`/api/getData?_id=${id} `);
+            const results = await fetchData(`/api/getData?_id=${testId} `);
```

Comment:
Making this change will break other codes. Also, I do not think we need this change. Please see comment in ReleaseSummary.jsx